app/main.py

```diff
@@ -2,0 +1,19 @@
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
```

Comment:
Here you're returning a string immediately when a `VaccineError` or `NotWearingMaskError` is raised. This means that the function will stop executing as soon as one of these errors is encountered. If you want to check all friends and count the total number of masks to buy, you should not return from the function immediately. Consider collecting all errors in a list and process them after the loop.